packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -47,7 +47,7 @@ const selectors = {
 	// corner. This addresses the bug where the post-publish panel is immediately
 	// closed when publishing with certain blocks on the editor canvas.
 	// See https://github.com/Automattic/wp-calypso/issues/54421.
-	viewButton: 'text=/View (Post|Page)/',
+	viewButton: 'a:text-matches("View (Post|Page)")',
```

Comment:
I think this selector is being updated in another one of my PRs too: https://github.com/Automattic/wp-calypso/pull/60864/files#diff-8aeb2361aa173b5616012d0e178c97eee06325bc9dd1b7a9570037ada0238659

Reply:
Nice! I made mine match so there won't be any merge conflicts either way!